### PR TITLE
Added alphabetical sorting to the list of installed extensions

### PR DESCRIPTION
**Pull request by Alroniks**
Work done under task https://oat-sa.atlassian.net/browse/BOSA-23

It adds sorting to this list:
TAO 3 4 0-sprint121

### CODE REVIEW

File: actions/class.ExtensionsManager.php [hunk 1]
```diff
@@ -148,6 +160,9 @@ public function disable()
     public function enable()
     {
         $this->assertIsDebugMode();
+
+        $this->getPsrRequest()->getQueryParams();
```
Comment (Yury-Razhkou) [inline actions/class.ExtensionsManager.php]: Why do we need this here?

Comment (Alroniks) [reply actions/class.ExtensionsManager.php]: Ops, forget to remove some experiments

**Comment by Alroniks** [comment]
I've rerun checks manually in Jenkins and it works. Seems it was an issue when the commit was sent but now tests are passing.

File: actions/class.ExtensionsManager.php [hunk 2]
```diff
@@ -35,24 +35,36 @@
...
+    /**
+     * @param array $extensions
+     *
+     * @return array
+     */
+    private function sortExtensionList(array $extensions = [])
+    {
+        usort($extensions, static function ($a, $b) {
+            /** @var common_ext_Extension $a */
+            /** @var common_ext_Extension $b */
```
Comment (wazelin) [inline actions/class.ExtensionsManager.php]: Better to set argument types of the anonymous function IMO.

Comment (wazelin) [reply actions/class.ExtensionsManager.php]: Oh, I was a couple of seconds late with that comment.

**Pull request merged**